modules/twinklespeedy.js

```diff
@@ -1409,7 +1409,7 @@ Twinkle.speedy.callbacks = {
 				return;
 			}
 
-			var xfd = /(?:\{\{([rsaiftcm]fd|md1|proposed deletion)[^{}]*?\}\})/i.exec( text );
+			var xfd = /(?:\{\{((?:article for deletion|template for discussion)\/dated|[cfm]fd|proposed deletion).*?\}\})/i.exec( text ) || /(#invoke:RfD)/.exec(text);
```

Comment:
Don't report back the whole `#invoke` method, just the name

```suggestion
			var xfd = /(?:\{\{((?:article for deletion|template for discussion)\/dated|[cfm]fd|proposed deletion).*?\}\})/i.exec( text ) || /(?:#invoke:(RfD))/.exec(text);
```